app/assets/v2/css/activity_stream.css

```diff
@@ -52,6 +52,63 @@
   text-align: left;
 }
 
+.hide {
```

Comment:
we have many classes for display:none already as we use bootstrap you can use d-none for example.